app/forms/base_work_form.rb

```diff
@@ -58,6 +61,8 @@ def deserialize!(params)
     deserialize_embargo(params)
     access_from_collection(params)
     deserialize_license(params)
+    # determine which version description field to use
```

Comment:
Can you move this down to the method?  Otherwise it sounds like a `TODO` rather than documentation of what the method docs.